components/menuData.ts

```diff
@@ -1828,13 +1831,82 @@ menuDataRouter.post(
     createLoadSaveMiddleware("SaveMenuTemplate"),
 )
 
-//TODO: Add statistics
 menuDataRouter.get("/PlayerProfile", (req: RequestWithJwt, res) => {
     const playerProfilePage = getConfig<PlayerProfileView>(
         "PlayerProfilePage",
         true,
     )
 
+    const locationData = getVersionedConfig<PeacockLocationsData>(
+        "LocationsData",
+        req.gameVersion,
+        false,
+    )
+
+    playerProfilePage.data.SubLocationData = []
+
+    for (const subLocationKey in locationData.children) {
+        //Ewww...
+        if (
+            subLocationKey === "LOCATION_ICA_FACILITY_ARRIVAL" ||
+            subLocationKey === "LOCATION_HOKKAIDO_SHIM_MAMUSHI" ||
+            subLocationKey.search("SNUG_") > 0
```

Comment:
```suggestion
            subLocationKey.includes("SNUG_")
```

Reply:
This was literally copied from your own logic, by the way, original I just had a startsWith :P

Reply:
https://github.com/thepeacockproject/Peacock/blob/5fff1c89ae739e063d713974a788af4bc38d82bb/components/menuData.ts#L240-L246

Reply:
Then I think both should use `includes`.